integration/jest.config.js

```diff
@@ -12,16 +12,18 @@ module.exports = {
   displayName: 'integration',
   preset: '../jest.preset.js',
   setupFilesAfterEnv: ['<rootDir>/test-setup.ts'],
-  globals: {
-    'ts-jest': {
-      tsconfig: '<rootDir>/tsconfig.spec.json',
-      stringifyContentPathRegex: '\\.(html|svg)$'
-    }
-  },
+  globals: {},
   moduleNameMapper,
   coverageDirectory: '../coverage/integration',
   transform: {
-    '^.+\\.(ts|js|html)$': 'jest-preset-angular'
+    '^.+\\.(ts|mjs|js|html)$': [
+      'jest-preset-angular',
+      {
+        isolatedModules: true,
+        tsconfig: '<rootDir>/tsconfig.spec.json',
+        stringifyContentPathRegex: '\\.(html|svg)$'
```

Comment:
Why was this needed?
If it was auto-generated by NX, then it's just their implementation details, but if it was something that was added by us for NGXS, then is this something we should add as guidance in our docs?

Reply:
Made by NX